### ISSUE TITLE
why should move the local /.env/lib/python3.11 to /cpython/lib/python3.12

### ISSUE THREAD

**Issue by guilhermeprokisch** [wasmerio/python-flask-example#1]
Hello there,

Just a noob question but why do I need to install the dependencies of the project in a virtual env and then associate the local binary to the "container" as in https://github.com/wasmerio/python-flask-example/blob/74cb87ad866332de019840212319b634df7b0c0d/wasmer.toml#L13

I was thinking more that the wasm file could contain all my depencies.

**Comment by syrusakbary** [comment]
This is because the Wasmer container is build locally (with the local python), and this is the path where our local Python (3.11) is installing the dependencies on the virtualenv @guilhermeprokisch

**Comment by guilhermeprokisch** [comment]
I see,  but somehow I could not have .wasm file from the project container?

**Comment by guilhermeprokisch** [comment]
And since wasix support sockets it's not possible to implement pip on it?

**Comment by syrusakbary** [comment]
Yup, pip should fully work there! (Although we haven’t tried yet)
